src/leo-client-app/src/components/EditSchedules.tsx

```diff
@@ -8,26 +8,38 @@ import "../styles.css";
 import "./styles/Scheduler.css";
 import SatelliteName from "./SatelliteName";
 import { Box, Card, Typography, Stack, Button } from "@mui/material";
+import axios from "axios";
 
-type Props = {
-  noradId: string;
-};
-
-const EditScheduler = ({ noradId }: Props) => {
+const EditScheduler = () => {
   const queryClient = useQueryClient();
   const router = useRouter();
+  const { satId, scheduleId } = router.query as {
+    satId: string;
+    scheduleId: string;
+  };
```

Comment:
can you instead do something like this

const satId = router.query.satId?.toString() ?? '';
const scheduleId = router.query. scheduleId?.toString() ?? '';

This way you won't have to cast a type